start-console-auth-mode.sh

```diff
@@ -4,9 +4,9 @@ set -euo pipefail
 PATH+=:/usr/bin
 
 oc process -f scripts/oauth-client.yaml | oc apply -f -
-oc get oauthclient console-oauth-client -o jsonpath='{.secret}' > scripts/console-client-secret
-oc get secrets -n default --field-selector type=kubernetes.io/service-account-token -o json | \
-    jq '.items[0].data."ca.crt"' -r | python3 -m base64 -d > scripts/ca.crt
+oc get oauthclient console-oauth-client -o jsonpath='{.secret}' >scripts/console-client-secret
```

Comment:
Is there a reason why you're removing the spaces after the angle brackets?

Reply:
its the auto save.. didn't touch it just left prettier fixes